app/views/content/life-as-a-teacher/explore-subjects/drama/_article.html.erb

```diff
@@ -0,0 +1,67 @@
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching drama opens the door for your pupils to pursue exciting careers in fields such as acting, stage management and set design.</p>
+      <p>Your pupils will learn more from you than you might think. As their drama teacher, you'll guide them through the skills needed to write, rehearse and perform dramatic pieces. You'll teach them valuable life skills such as confidence with public speaking, working collaboratively and thinking creatively.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_drama_itasha' %>
+  </section>
+
+  <section class="col col-720">
+   <h2 class="heading--box-blue">What you'll teach</h2>
+   <p>Drama is part of the spoken language element of the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-english-programmes-of-study/national-curriculum-in-england-english-programmes-of-study">national curriculum for English</a>.</p>
+   <p>When you teach 11 to 16 year olds (key stages 3 and 4), you might cover skills including:</p>
+    <ul>
+      <li>improvising and rehearsing scripts</li>
+      <li>discussing language use and meaning</li>
+      <li>using intonation, voice and tone to add impact</li>
+      <li>using space effectively</li>
+    </ul>
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+    <ul>
+      <li>the theatrical processes and practices involved in interpreting and performing theatre</li>
+      <li>how conventions, forms and techniques are used in drama and live theatre to create meaning</li>
+      <li>how creative and artistic choices influence how meaning is communicated to an audience </li>
+      <li>how performance texts are constructed to be performed</li>
+      <li>how performance texts are informed by their social, cultural and historical contexts, and are interpreted and performed for an audience</li>
+    </ul>
+    <%= render 'content/shared/teaching/ages_abilities' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching drama</h2>
+    <%= render 'content/shared/qualifications-training/check_qualifcations' %>
+    <p>Drama teacher training courses have had successful applications from candidates with a range of degrees, including dance, film studies, English and law.</p>
+    <%= render 'content/shared/qualifications-training/find_out_more' %>
+    <%= render 'content/shared/qualifications-training/non_uk_qualifications' %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <!-- there will be a partial for no funding when history goes out -->
+    <%= render 'content/shared/teaching/funding_support' %>
```

Comment:
Is this partial missing here? Think it should start with the bit about bursaries?